conf/openlibrary-docker.yml

```diff
@@ -0,0 +1,208 @@
+
+connection_type: hybrid
+
+site: openlibrary.org
+
+from_address: "Open Library <noreply@openlibrary.org>"
+report_spam_address: "Open Library <info@openlibrary.org>"
+
+support_case_notification_address: "Open Library <info@openlibrary.org>"
+support_case_control_address : "support@openlibrary.org"
+
+# Legal values for support_default_assignees are
+#
+# bug_report
+# borrowing_books
+# buying_books
+# contact_an_author
+# deletion_request
+# developer/api/covers_question
+# editing_problem
+# image_upload
+# login_trouble
+# spam_report
+# other
+#
+# and
+#
+# default (in case there is no special email address for the given category)
+
+support_default_assignees:
+  developer/api/covers_question/bug_report: "noufal+bugs@gmail.com"
+  default: "noufal+default@gmail.com"
+
+
+smtp_server: localhost
+
+dummy_sendmail: True
+debug: True
+
+coverstore_url: http://0.0.0.0/covers/
+
+state_dir: var/run
+
+# enable http compression
+use_gzip: True
+
+admin_password: admin123
+errorlog: /var/log/openlibrary/ol-errors
+login_cookie_name: session
+
+infobase_server: localhost:7000
+
+# contents of this file are assigned to config["infobase"]
+# Path can be relative to this file
+infobase_config_file: infobase.yml
+coverstore_config_file: coverstore.yml
+
+loanstatus_url: http://localhost/
+ia_access_secret: foobar
+
+#dev_instance: True
+dev_instance_url: http://127.0.0.1:8080/
+
+memcache_servers:
+    - localhost:11211
+
+plugin_modules:
+    - openlibrary
+    - infogami.plugins.links
+    - infogami.plugins.api
+
+plugin_worksearch:
+    solr: solr:8080
+    spellcheck_count: 3
+    ebook_count_db_parameters:
+        db: openlibrary_ebook_count
+        host: localhost
+
+ia_ol_xauth_s3:
+    s3_key: XXX
+    s3_secret: XXX
+
+ia_loan_api_url: http://127.0.0.1/internal/fake/loans
+ia_availability_api_url: http://127.0.0.1/internal/fake/availability
+ia_xauth_api_url: http://127.0.0.1/internal/fake/xauth
+http_request_timeout: 10
+
+stats_solr: solr:8080
+
+features:
+    upstream: enabled
+    cache_most_recent: enabled
+    recentchanges_v2: enabled
+    history_v2: admin
+    merge-authors:
+        filter: usergroup
+        usergroup: /usergroup/librarians
+    merge-editions: admin
+    undo: enabled
+    dev: enabled
+    lists: enabled
+    lending_v2: enabled
+    stats: enabled
+    stats-header: enabled
+    inlibrary: enabled
+    support: admin
+    dev: enabled
+    superfast: enabled
+    publishers: enabled
+    languages: enabled
+
+upstream_to_www_migration: true
+default_template_root: /upstream
+css_root: /upstream/css
+js_root: /upstream/js
+use_google_cdn: false
+logging_config_file: conf/logging.ini
+email_config_file: conf/email.ini
+
+lists:
+    seeds_db: http://127.0.0.1:5984/seeds
+    editions_db: http://127.0.0.1:5984/editions
+    works_db: http://127.0.0.1:5984/works
+    editions_view: http://127.0.0.1:5984/editions/_fti/_design/seeds/by_seed
+
+admin:
+    olsystem_root: /home/noufal/projects/OL/olsystem/
```

Comment:
I think this can be pruned?

Reply:
It is used in code here: https://github.com/internetarchive/openlibrary/blob/75452cf6f0e3c23e1030395c93e232686ebaa417/openlibrary/plugins/admin/code.py#L538  but I can't get the page `/admin/services` to display. Looks like it is not registered in the `setup()` further down in that file.   Shall I prune the entire `service_status` class?